tools/metadatastore-upgrade/README.md

```diff
@@ -0,0 +1,34 @@
+# MetadataStore Upgrade Tool
+
+Upgrade tool provides a mechanism for KFP users to upgrade MetadataStore component in their KFP cluster. A MetadatatStore upgrade is composed of two aspects:
+* Upgrading the image used in `metadata-deployment` K8s Deployment. The user is expected to provide an image-tag for this upgrade.
+* Upgrading the MYSQL database schema to adhere to MLMD library used in the `metadata-deployment` image. The tool automatically handles schema upgrade using a [feature](https://github.com/google/ml-metadata/releases/tag/v0.21.0) in the gRPC server used in the `metadata-deployment` image. 
```

Comment:
Do you mean to say the gRPC server handles the schema migration of the backing mySQL database if some flag is set to true? If so, maybe reword this sentence to better reflect that, and link to the flag definition?

Reply:
Done.